lesson7-questions.txt

```diff
@@ -43,7 +54,7 @@ has_many :forums, through: :subscriptions
 What are threee important methods that are available to instances of the user model because
 of these lines?
 
-
+it is associated and takes attributes from subscriptions, posts, and forums
```

Comment:
just want to point out that because these are associated through `has_many`, that these methods will bring up arrays. the answer was right, but wanted to differentiate it from #4